src/js/routes/services.js

```diff
@@ -13,7 +13,7 @@ let serviceRoutes = {
   children: [
     {
       type: Route,
-      name: 'services-deployments',
+      name: '-services-deployments',
```

Comment:
One option is just to name it 'deployments', which I'm also OK with - it makes it seem like it might be a left-hand menu route though, which it isn't.